build/Autofac.Build.psm1

```diff
@@ -87,6 +87,17 @@ function Install-DotNetCli
   $env:PATH = "$installDir;$env:PATH"
 }
 
+<#
+ .SYNOPSIS
+  Installs ReportGenerator CLI
+  https://github.com/danielpalme/ReportGenerator
+#>
+function Install-ReportCliTool
+{
```

Comment:
Is it worth defining the coverage report tool as a local tool rather than global? Then we can just call `dotnet tool restore` instead of installing the tool in the script.

Reply:
I actually initially tried that using a very similar logic like the function `Install-DotNetCli`.
Sadly, the tool was not added to the path, even though I called this:

```powershell
$env:PATH = "$installDir;$env:PATH"
```

Not sure why it didn't work though using Power-Shell. I am also not sure if it really makes a difference. What do you think?

Reply:
The local tools don't go on the path as-is.

You'll need to do `dotnet <tool name>` to run the tool.

Reply:
> You'll need to do `dotnet <tool name>` to run the tool.

That didn't work as well. I ended up doing something different but also only with a local installation.